### ISSUE TITLE
Request: Odin Programming Language

### ISSUE THREAD

**Issue by krios2146** [simple-icons/simple-icons#12133]
### Brand Name

Odin Programming Language

### Website

https://odin-lang.org/

### Popularity Metric

The Similarweb global rank is 447,974. See: https://www.similarweb.com/website/odin-lang.org/

### Official Resources for Icon and Color

Logo PNG: https://github.com/odin-lang/Odin/blob/master/misc/logo-slim.png

Color is probably dark blue - HSL(211, 63%, 32%) which is one of the https://odin-lang.org/ background colors

### Additional Comments

Language repository has BSD-3 license, no explicit license for logo, but I assume it's the same

**Comment by uncenter** [comment]
The Similarweb rank is just barely in our criteria, but the repository has enough GitHub stars and is plenty active: https://github.com/odin-lang/Odin.

Logo at https://github.com/odin-lang/artwork/blob/master/logo/emblem-without-background.svg is what we should use for the source. The hex for that HSL is `#1E5085`, which looks like a good color to use.

**Comment by Anonav0** [comment]
so should I use this  https://github.com/odin-lang/artwork/blob/master/logo/emblem-without-background.svg as the logo? 

I want to contribute to this project.

**Comment by uncenter** [comment]
> so should I use this [odin-lang/artwork@master/logo/emblem-without-background.svg](https://github.com/odin-lang/artwork/blob/master/logo/emblem-without-background.svg) as the logo?

Yep, as I said in my previous comment.

**Comment by Anonav0** [comment]
Alright.

**Comment by Anonav0** [comment]
> > so should I use this [odin-lang/artwork@master/logo/emblem-without-background.svg](https://github.com/odin-lang/artwork/blob/master/logo/emblem-without-background.svg) as the logo?
> 
> Yep, as I said in my previous comment.

Hey I'm trying to add this link as the source of the logo in the JSON file. but running the lint is asking me to provide something else as a link. Any solution to that? Test was successful locally.

**Comment by uncenter** [comment]
As per https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines:

> The commit hash should always be part of the URL. On GitHub, you can get the correct URL by pressing y on the GitHub page you want to link to. You can get help at the [getting permanent links to files page](https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files).

I think that should be at least mentioned in the error you are getting? Let me know if not.

**Comment by Anonav0** [comment]
yeah! Thanks for the help. It's working now. Gonna create a PR soon. 🙌🏾
